client/src/document/hooks.ts

```diff
@@ -13,20 +14,23 @@ export function useDocumentURL() {
 
 export function useCopyExamplesToClipboard(doc: Doc | undefined) {
   const location = useLocation();
+  const isServer = useIsServer();
 
   useEffect(() => {
-    if (!doc) {
-      return;
-    }
-    if (!navigator.clipboard) {
-      console.log(
-        "Copy-to-clipboard disabled because your browser does not appear to support it."
-      );
-      return;
-    }
+    if (!isServer) {
```

Comment:
To improve readability, let's return early:

```suggestion
    if (isServer) {
      return;
    }

```

Reply:
ah you're right, it looks better this way. thank you <3.